ui/admin/tests/acceptance/host-catalogs/hosts/read-test.js

```diff
@@ -87,26 +82,28 @@ module('Acceptance | host-catalogs | hosts | read', function (hooks) {
     instances.host.authorized_actions =
       instances.host.authorized_actions.filter((item) => item !== 'read');
 
-    await click(`[href="${urls.hosts}"]`);
-
-    assert.dom('.rose-table-body  tr:first-child a').doesNotExist();
+    await click(commonSelectors.HREF(urls.hosts));
+    assert.dom(commonSelectors.TABLE_FIRST_ROW_RESOURCE_LINK).doesNotExist();
```

Comment:
non-blocking: Could we add the space between the click and assert? 
```suggestion
    await click(commonSelectors.HREF(urls.hosts));
    
    assert.dom(commonSelectors.TABLE_FIRST_ROW_RESOURCE_LINK).doesNotExist();
```